.travis.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
It looks like Travis doesn't execute the unit tests:
```
[WARN]  No unit tests executed.
```
which makes the CI build basically pointless. Or am I missing something here?

Reply:
I see following lines:

```
[WARN]  No unit tests executed.
[INFO]  All unit tests passed.
```

So, it seems like unit tests are actually executed right? I check on local and even if it says No unit tests executed in first line, all unit tests are actually getting executed.

Reply:
Have you tried letting a unit test fail and see whether this lets the Travis build fail?

A quick Google search resulted in this [SO post](https://stackoverflow.com/questions/49765661/pybuilder-no-unit-tests-executed) which links to the [PyBuilder docs](http://pybuilder.github.io/documentation/tutorial.html) which mention the requirements to find tests:

> Concerning the filename it is important to notice that

>The file must be located under src/unittest/python. This is the default and may be altered but as with the main sources, we stick to the defaults.
>The file must end with _tests.py. This tells PyBuilder to consider this file when discovering test cases. Again, this can be altered, although we don’t do this.

Your files end on `_test.py` instead of `_tests.py` and are in a different directory. Is that maybe the problem?